fiftyone/core/collections.py

```diff
@@ -3339,6 +3339,21 @@ def compute_metadata(
             progress=progress,
         )
 
+    def compute_annotation_schema(self, field_name):
+        """Compute the annotation schema for a field
```

Comment:
For consistency with existing `SampleCollection` methods and to maximize usability for SDK users, I'd like us to include a full copy of the docstring from `foa.compute_annotation_schema()` here.